base/applications/games/winmine/main.c

```diff
@@ -653,15 +663,19 @@ static void CompleteBox( BOARD *p_board, unsigned col, unsigned row )
         if( p_board->box[col][row].IsMine ) {
             p_board->face_bmp = DEAD_BMP;
             p_board->status = GAMEOVER;
+            PlaySoundEffect(IDW_EXPLODE);
         }
-        else if( p_board->status != GAMEOVER )
+        else if( p_board->status != GAMEOVER ) {
             p_board->boxes_left--;
+            if (is_first_box)
+                PlaySoundEffect(IDW_BOX);
+        }
```

Comment:
The code is winesynced, except the resources: https://github.com/reactos/reactos/blob/c3ec1b9ac63ec628f49d8e10c0c1d7950bb26c5a/media/doc/WINESYNC.txt#L248

Reply:
I doubt Wine maintains this anymore, however. Same with WordPad and other little utilities like this.
If someone really wanted to fork it, I'd approve it; but I don't see the value in doing that at the moment.

Reply:
Fixed.